client/components/User/UserPage.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <main>
+    <section>
+      <ChangeUsernameForm />
+      <ChangePasswordForm />
+    </section>
+    <hr/>
+    <section>
+      <LogoutForm />
+    </section>
+    <hr/>
+  </main>
+</template>
+
+<script>
+import ChangeUsernameForm from '@/components/User/ChangeUsernameForm.vue';
+import ChangePasswordForm from '@/components/User/ChangePasswordForm.vue';
+import LogoutForm from '@/components/User/LogoutForm.vue';
+
+export default {
+  name: 'UserPage',
+  components: {
+    ChangeUsernameForm,
+    ChangePasswordForm,
+    LogoutForm,
+  },
+  mounted() {
+    this.$store.commit('setHeader', {
+      title: `Login Settings for ${this.$store.getters.username}`,
```

Comment:
This is extremely nitpicky but reactivity inconsistent

![image](https://user-images.githubusercontent.com/59969293/206896631-8d264c6f-d217-4c6a-a0f9-2295886a7314.png)

Reply:
good catch!